src/Validation/test/Microsoft.Extensions.Validation.GeneratorTests/snapshots/ValidationsGeneratorTests.CanUseAutoGeneratedValidatableTypeAttribute#ValidatableTypeAttribute.g.verified.cs

```diff
@@ -0,0 +1,9 @@
+﻿//HintName: ValidatableTypeAttribute.g.cs
+// <auto-generated/>
+namespace TestApp;
```

Comment:
This namespace looks incorrect. Is it because the source strings in the test code use file-scoped namespaces? 🤔

Reply:
I regenerated the tests. For some reason the tests were passing but nothing was happening. I even introduced a compilation issue on purpose to make them fail and were green.

I switched to use the old style namespaces and Program main and that got them "working" again